src/components/habit/habit-today/TodayHabitItems.tsx

```
@@ -0,0 +1,75 @@
+// "use client";
+import { API_URL } from "@/constants";
+import fetchData from "@/lib/apis/fetchData";
+import { Habit, habitStatus, FK, PK } from "@/types";
```

ESLint

[error] 'habitStatus' is defined but never used.

(@typescript-eslint/no-unused-vars)